src/scripts/main.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const parentList = document.getElementById('thumbs');
+
+parentList.addEventListener('click', (evt) => {
+  if (evt.target) {
+    evt.preventDefault();
+    largeImg.setAttribute('src', evt.target.src);
+  }
+});
```

Comment:
```suggestion
parentList.addEventListener('click', (event) => {
  event.preventDefault();

  const link = event.target.closest('.list-item__link');

  largeImg.src = link.href;
});
```
to fix the image in demo link